packages/cart/src/repositories/address.ts

```diff
@@ -0,0 +1,24 @@
+import { Context } from "@medusajs/types"
+import { DALUtils } from "@medusajs/utils"
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+import { Address } from "@models"
+import { UpdateAddressDTO } from "../types"
+
+export class AddressRepository extends DALUtils.mikroOrmBaseRepositoryFactory<Address>(
+  Address
```

Comment:
todo: missing create dto type here

Reply:
Why do I need a create DTO?

Reply:
By default, the base class does not know what DTO to apply for the mutation methods, so they are any. If you want to get the DTO types applied when consuming the repository, we need to provide it to the factory to apply it. here the update is overridden so the DTO is known, but the create is not. wdyt?

Reply:
I see. I'll add that in 👍